Recipe-App/Recipe-App/UIViews/Components/ImageTitleRatingCardView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ImageTitleRatingCardView.swift
+//  Recipe-App
+//
+//  Created by Gontse Ranoto on 2021/06/14.
+//
+
+import SwiftUI
+
+struct ImageTitleRatingCardView: View {
```

Comment:
Can we rename this to RecipeCardItemView item

Reply:
@MicahSphelele  Done!